Makefile

```diff
@@ -53,7 +53,7 @@ install:
 
 .PHONY: lint
 lint:
-	which golint || $(GOGET) -u golang.org/x/lint/golint
+	which golint || GO111MODULE=off $(GOGET) -v -u golang.org/x/lint/golint
```

Comment:
why should set `GO111MODULE=off` on this?  😕

Reply:
If you don't turn it off temporarily, dependencies will be added to go mod.

```
❯ make lint
go: finding github.com/KyleBanks/depth v1.2.1
go: finding github.com/ghodss/yaml v1.0.0
go: finding github.com/swaggo/gin-swagger v1.2.0
go: finding github.com/swaggo/files v0.0.0-20190704085106-630677cd5c14
go: finding github.com/stretchr/testify v1.3.0
go: finding github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
go: finding github.com/go-openapi/spec v0.19.0
go: finding github.com/go-openapi/jsonreference v0.19.0
go: finding github.com/urfave/cli v1.20.0
go: finding golang.org/x/tools v0.0.0-20190611222205-d73e1c7e250b
go: finding github.com/stretchr/testify v1.2.2
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/pmezard/go-difflib v1.0.0
go: finding github.com/go-openapi/swag v0.17.0
go: finding github.com/go-openapi/jsonpointer v0.17.0
go: finding github.com/mailru/easyjson v0.0.0-20180823135443-60711f1a8329
go: finding gopkg.in/yaml.v2 v2.2.1
```